.github/workflows/size-limit.yml

```diff
@@ -1,10 +1,6 @@
 name: Size Testing
 
-on:
-  pull_request_target:
-    types:
-      - opened
-      - synchronize
+on: [pull_request]
```

Comment:
Isn't `on: pull_request` enough? The array should be used for multiple events [according to docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#using-multiple-events).